dev-docs/bidder-adaptor.md

```diff
@@ -12,259 +11,504 @@ nav_section: adapters
 
-In your PR to add the new adapter, please provide the following information:
+### Required adapter conventions
 
-- The contact email of the adapter's maintainer.
-- A test ad unit that will consistently return test creatives. This helps us to ensure future Prebid.js updates do not break your adapter.
-- Any other information listed as required in [CONTRIBUTING.md](https://github.com/prebid/Prebid.js/blob/master/CONTRIBUTING.md).
+Adapters must follow the conventions listed below.
 
-## Step 2: Add a new bidder JS file
+In order to provide a fast and safe header bidding environment for publishers, the Prebid.org team reviews all adapters for the following required conventions:
 
-1. Create a JS file under `modules` with the name of the bidder suffixed with 'BidAdapter', e.g., `xyzBidAdapter.js`
+* *Support multiple instances*: All adapters must support the creation of multiple concurrent instances. This means, for example, that adapters cannot rely on mutable global variables.
+* *No loading of external libraries*: All code must be present in the adapter, not loaded at runtime.
+* *Must support HTTPS*: Within a secure page context, the request to the bidder's server must also be secure.
+* *Compressed responses*: All bid responses from the bidder's server must be gzipped.
+* *Bid responses may not use JSONP*: All requests must be AJAX with JSON responses.
+* *All user-sync activity must be registered via the provided functions*: The platform will place all registered syncs in the page after the auction is complete, subject to publisher configuration.
+* *Adapters may not use the `$$PREBID_GLOBAL$$` variable*: Instead, they must load any necessary functions and call them directly.
 
-2. Create an adapter factory function with the signature shown below:
+{: .alert.alert-danger :}
+Failure to follow any of the above conventions could lead to delays in approving your adapter for inclusion in Prebid.js.
 
-{% highlight js %}
-import bidfactory from 'src/bidfactory';
-import bidmanager from 'src/bidmanager';
-import * as utils from 'src/utils';       // useful functions
-import { ajax } from 'src/ajax';          // recommended AJAX library
-import { STATUS } from 'src/constants';
-function xyzBidAdapter() {
-
-    function _callBids(params) {... add code here ...}
-
-    // Export the `callBids` function, so that Prebid.js can execute
-    // this function when the page asks to send out bid requests.
-    return {
-        callBids: _callBids
-    };
-};
+### Design your bid params
 
-adaptermanager.registerBidAdapter(new XYZBidAdapter, 'xyz');
-module.exports = xyzBidAdapter;
-{% endhighlight %}
+The AdUnit's `bid.params` object will define the parameters of your ad request.  You can include tag ID, site ID, ad size, keywords, and other data, such as video parameters.
+
+For more information about the kinds of information that can be passed using these parameters, see [the existing bidder parameters]({{site.baseurl}}/dev-docs/bidders.html).
```

Comment:
Wording seems a bit awkward.

Reply:
Agreed, esp. that first sentence.  Tweaked in 838ac64